tools/cloud-build/daily-tests/builds/hcls-v6.yaml

```diff
@@ -34,6 +34,22 @@ tags:
 
 timeout: 14400s  # 4hr
 steps:
+# While using static network names we are gaurding against more than 1 instance running at a time (for multi-group tests)
+- id: check_for_running_build
+  name: gcr.io/cloud-builders/gcloud
+  entrypoint: /bin/bash
+  args:
```

Comment:
nit. Let's move it into a shell script, do avoid copy pasting it.

Reply:
Good idea, will do in follow on PR just to get this in ASAP.